var/spack/repos/builtin/packages/cdo/package.py

```diff
@@ -160,14 +160,14 @@ class Cdo(AutotoolsPackage):
     depends_on("netcdf-c", when="+netcdf")
     # The internal library of CDO implicitly links to hdf5.
     # We also need the backend of netcdf to be thread safe.
-    depends_on("hdf5+threadsafe", when="+netcdf")
+    depends_on("hdf5+threadsafe", when="+netcdf +openmp")
 
     depends_on("grib-api", when="grib2=grib-api")
     depends_on("eccodes", when="grib2=eccodes")
 
     depends_on("szip", when="+szip")
 
-    depends_on("hdf5+threadsafe", when="+hdf5")
+    depends_on("hdf5+threadsafe", when="+hdf5 +openmp")
```

Comment:
Why is this line repeated here? Same as 163?

Reply:
I guess they decided to cover the `+netcdf` and `hdf5` cases with two separate lines. I suppose it could be condensed to `when="+openmp ^hdf5"` but I wanted to minimize other changes.